js/LayerList.js

```diff
@@ -97,7 +98,7 @@ const LayerList = (() => {
                 toMergeImageData.data[i+2], toMergeImageData.data[i+3]
             ];
 
-            let currentUnderlyingPixel = [
+            let currentUnderlyingPixel = [ //TODO: I'd be curious to know if this array slows this function down
```

Comment:
For big sprites, probably. currentUnderlyingPixel shouldn't be modified iirc, so we can just use belowImageData.data I guess